src/main/java/hudson/plugins/parameterizedtrigger/TriggerBuilder.java

```diff
@@ -151,7 +154,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
                                     BuildInfoExporterAction.addBuildInfoExporterAction(build, completedRun.getParent().getFullName(), completedRun.getNumber(), completedRun.getResult());
 
                                     if (buildStepResult && config.getBlock().mapBuildStepResult(completedRun.getResult())) {
-                                        build.setResult(config.getBlock().mapBuildResult(completedRun.getResult()));
+                                        Result r = config.getBlock().mapBuildResult(completedRun.getResult());
+                                        if (r != null) {
```

Comment:
`r == null ` when the run is not finished.
Maybe it makes sense to add a system warning for the edge case

Reply:
Sure I can add it in case there is some underlying issue here